forge/src/main/java/org/openjdk/skara/forge/github/GitHubRepository.java

```diff
@@ -145,13 +145,26 @@ public PullRequest pullRequest(String id) {
 
     @Override
     public List<PullRequest> pullRequests() {
-        return request.get("pulls").execute().asArray().stream()
+        return request.get("pulls")
+                      .param("state", "all")
+                      .execute().asArray().stream()
+                      .map(jsonValue -> new GitHubPullRequest(this, jsonValue, request))
+                      .collect(Collectors.toList());
+    }
+
+    @Override
+    public List<PullRequest> openPullRequests() {
+        // The default `state` parameter is `open` in GitHub which is not same as `GitLab`.
+        // Here, the `state` parameter is also added to avoid misunderstanding.
```

Comment:
I don't think we need this comment. Same for lines 181:182.

Reply:
Removed.